### ISSUE TITLE
CSS warnings appear in the console

### ISSUE THREAD

**Issue by not-an-aardvark** [porygonco/porybox#197]
> ARIA: Attribute " aria-label ", required for accessibility, is missing on node:

**Comment by YaManicKill** [comment]
I think I've fixed this now (on the profile branch). Let me know if you see anymore

**Comment by not-an-aardvark** [comment]
I'm still getting some of the same warnings on the login page.
